File: PRAK204-2210817120007-NursyahnaPuteri.py
jarijaribejana = float (input("masukan jari-jari dari bejana: "))
tinggibejana = float (input("masukan tinggi dari bejana : "))
phi = 3.14285714

volume = phi * jarijaribejana* jarijaribejana * tinggibejana
luas = 2 * phi * jarijaribejana * (jarijaribejana + tinggibejana)
keliling = 2 * phi * jarijaribejana

print ("Volume = {:.2f} ".format (volume))
print ("Luas =  {:.2f} ".format (luas))
print ("Keliling = {:.2f}".format (keliling))

File: PRAK204-2210817120007-NursyahnaPuteri.c
#include <stdio.h>
int main ()
{
    float jarijari, tinggi, volume, luas, keliling;
    const float phi = 3.14285714;

    printf ("Masukkan nilai jari-jari bejana : ");
    scanf ("%f", &jarijari);

    printf ("Masukkan nilai tinggi bejana : ");
    scanf ("%f", &tinggi);

    printf ("\n\n");

    volume = phi * jarijari * jarijari * tinggi;
    luas = 2 * phi * jarijari * (jarijari + tinggi);
    keliling = 2 * phi * jarijari ;
    
    printf (" Volume =  %.2f\n", volume);
    printf (" Luas =  %.2f\n", luas);
    printf (" keliling =  %.2f\n", keliling );
    printf ("\n\n");
   
}